bot/exts/fun/off_topic_names.py

```diff
@@ -117,25 +152,134 @@ async def delete_command(self, ctx: Context, *, name: OffTopicName) -> None:
         log.info(f"{ctx.author} deleted the off-topic channel name '{name}'")
         await ctx.send(f":ok_hand: Removed `{name}` from the names list.")
 
-    @otname_group.command(name='list', aliases=('l',))
+    @otname_group.command(name='activate', aliases=('whitelist',))
+    @has_any_role(*MODERATION_ROLES)
+    async def activate_ot_name(self, ctx: Context, name: OffTopicName) -> None:
+        """Activate an existing off-topic name."""
+        await self.toggle_ot_name_activity(ctx, name, True)
+
+    @otname_group.command(name='deactivate', aliases=('blacklist',))
+    @has_any_role(*MODERATION_ROLES)
+    async def de_activate_ot_name(self, ctx: Context, name: OffTopicName) -> None:
+        """Deactivate a specific off-topic name."""
+        await self.toggle_ot_name_activity(ctx, name, False)
+
+    @otname_group.command(name='reroll')
+    @has_any_role(*MODERATION_ROLES)
+    async def re_roll_command(self, ctx: Context, ot_channel_index: Optional[int] = None) -> None:
+        """
+        Re-roll an off-topic name for a specific off-topic channel and deactivate the current name.
+
+        ot_channel_index: [0, 1, 2, ...]
+        """
+        if ot_channel_index:
```

Comment:
You can't actually reroll off-topic channel 0 because it is falsely.
```suggestion
        if ot_channel_index is not None:
```